core/src/main/scala/io/chrisdavenport/shellfish/syntax/path/package.scala

```diff
@@ -0,0 +1,374 @@
+     * @param path
+     *   The path to write to
+     * @param contents
+     *   The contents to write to the file
+     * @return
+     *   The number of bytes written
+     */
+    def writeLines(contents: Vector[String]): IO[Long] =
+      FilesOs.writeLines(path)(contents)
+
+    /**
+     * The functions writes the contents of the file at the path with the
+     * contents provided and returns the number of bytes written. The codec is
+     * used to translate the type A into a ByteVector so it can be parsed into
+     * the file.
+     *
+     * @param A
+     *   The type of the contents to write
+     * @param path
+     *   The path to write to
+     * @param contents
+     *   The contents to write to the file
+     * @param codec
+     *   The codec that translates the type A into a ByteVector
+     * @return
+     *   The number of bytes written
+     */
+    def writeAs[A](contents: A)(implicit codec: Codec[A]): IO[Long] =
+      FilesOs.writeAs(path)(contents)
+
+    /**
+     * Similar to `write`, but appends to the file instead of overwriting it.
+     * Saves the content at the end of the file in form of a String.
+     *
+     * @param path
+     *   The path to write to
+     * @param contents
+     *   The contents to write to the file
+     * @return
+     *   The number of bytes written
+     */
+    def append(contents: String): IO[Long] = FilesOs.append(path)(contents)
+
+    /**
+     * Similar to `write`, but appends to the file instead of overwriting it.
+     * Saves the content at the end of the file in form of a ByteVector.
+     *
+     * @param path
+     *   The path to write to
+     * @param contents
+     *   The contents to write to the file
+     * @return
+     *   The number of bytes written
+     */
+    def appendBytes(contents: ByteVector): IO[Long] =
+      FilesOs.appendBytes(path)(contents)
+
+    /**
+     * Similar to `write`, but appends to the file instead of overwriting it.
+     * Saves each line of the content at the end of the file in form of a Vector
+     * of Strings.
+     *
+     * @param path
+     *   The path to write to
+     * @param contents
+     *   The contents to write to the file
+     * @return
+     *   The number of bytes written
+     */
+    def appendLines(contents: Vector[String]): IO[Long] =
+      FilesOs.appendLines(path)(contents)
+
+    /**
+     * Similar to `write`, but appends to the file instead of overwriting it.
+     *
+     * @param A
+     *   The type of the contents to write
+     * @param path
+     *   The path to write to
+     * @param contents
+     *   The contents to write to the file
+     * @param codec
+     *   The codec that translates the type A into a ByteVector
+     * @return
+     *   The number of bytes written
+     */
+    def appendAs[A](contents: A)(implicit codec: Codec[A]): IO[Long] =
+      FilesOs.appendAs[A](path)(contents)
+
+  }
+
+  private val files = Files[IO]
+
+  implicit class FileOps(val path: Path) extends AnyVal {
```

Comment:
I'm not sure why you've ported just some methods from `Files[IO]` to `FilesOs`, while you ported the rest of the methods in this syntax class. I'm not telling is a bad thing, I'm interested in the design choice / the rationale you used to decide which to add there and which to add here.

Reply:
There's also something that I don't know. Given that we have 2 implicit classes on the same type in the same package, how would they behave in case we import everything with 
`import io.chrisdavenport.shellfish.syntax.path.*` ?

Reply:
Oh its because I thought that the methods in `Files[IO]` are already implemented, so I dont know how good would be to re-alias them, thats why I leaved the `val files = Files[IO]` not private so the user can call them. But now that you mention it, is it posible to do something similar to `export files.*` in Scala 2?

Reply:
Regarding the second question, how I experimented it is that it is behaving like a single extension method 🤔 I haven't exerienced bugs so far doing that but I don't know if that has some hidden implications that Im not aware of 🙃

Reply:
Following the comments some of us made in multiple locations (Discord, the main discussion on the PR and so) I'll say let's merge them